nasa/harmony
# DAS-1746 - Update package-lock.json with npm audit fixes.

## Jira Issue ID

DAS-1746 (well, it's a discovered while testing kind of thing)

## Description

@flamingbear and I have made a couple of changes to the `harmony-regression-tests` repository. Currently, testing in Bamboo is blocked because the main Harmony repository doesn't pass `npm audit`, and so the main Harmony build plan fails. This PR commits changes to the `package-lock.json` for Harmony, `query-cmr`, `service-runner` and `giovanni-adapter` to try and fix up those audit issues.

## Local Test Steps

* Pull this branch.
* Build Harmony locally.
* Ensure Harmony runs locally.

## PR Acceptance Checklist
* ~~Acceptance criteria met~~
* ~~Documentation updated (if needed)~~
